recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [win or osx]
+  skip: True  # [win]
```

Comment:
Do you know what went wrong here? https://circleci.com/gh/conda-forge/fish-feedstock/6?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

I just ran `conda smithy rerender` and then committed the change.